### ISSUE TITLE
GP classification description

### ISSUE THREAD

**Issue by benmarlin** [blei-lab/edward#244]
The description of GP classification here: http://edwardlib.org/tut_gp_classification seems to be incorrect. The description looks like Bayesian Logistic regression where the prior covariance matrix on the weights has been replaced with the Gram matrix of a GP covariance function. The dimensionality of z implied by p(z)​​​=N(z;0,K)​​ with K the Gram matrix is the number of data cases. x_n is stated to have dimension D. In that case, the inner product x​_n​'​​z in the inverse logit has a dimension mismatch. Compare to equation 3.10 in Rasmussen and Williams 2006 for their presentation of the posterior predictive distribution. In your notation, you probably want something like p(y​_n​​|x​_n,f​​)​​​=Bernoulli(y​n​​ | logit​inv​​(f(x​_n)​)) with f the draw from GP, or p(y​_n​​|x​_n,z_n)​​​=Bernoulli(y​n​​ | logit​inv​​(z​_n)​) with z_n = f(x​_n).

**Comment by dustinvtran** [comment]
Good catch! Yes you're right, the prior is written incorrectly, with the most obvious reason being that there's a dimension mismatch.
